021-IdentityForApps/Student/05-claims-enrichment.md

```diff
@@ -64,7 +65,7 @@ Your UserFlow will update all claims that are returned (such as TerritoryName) a
 CMC IT Leadership considers success in dealing with this QA issue if you are able to:
 
 - Create the new custom attribute for Territory Name in your B2C tenant (named "TerritoryName");
-- Deploy the Azure function (the "CMC ID Verify-inator"), code located in [Resources](./Resources/Verify-inator) ;
+- Deploy the "CMC ID Verify-inator" code located in [Resources](./Resources/Verify-inator);
```

Comment:
Please remove direct link to the WTH repo.  Instead, refer to the relative path in the Resources.zip file provided by the coach.

Reply:
Fixed!